### ISSUE TITLE
Check again that you do not use junit Assert, but only class azzert

### ISSUE THREAD

**Issue by dormaayan** [TechnionYP5779/team1#76]
_From @yossigil on November 14, 2018 9:14_

You can do this manually, or write a script. it is up to you. 

_Copied from original issue: TechnionYP5779/team6#79_

**Comment by ofiralexi** [comment]
In issue #62 we showed that the spartanizer wants to change azzert into assert. You told us to change back to assert . Clarification ? @dormaayan @yossigil

**Comment by ShaiZeevi** [comment]
Duplicate of #43

**Comment by ofiralexi** [comment]
@yoavz1997 Applied all azzerts and spartanized tips -> gets the code.
What do you think ? can I close?

**Comment by yoavz1997** [comment]
Yes, there are no more tips

**Comment by yossigil** [comment]
@SuppressWarnings("static-method") public class singletonTest {
  @Test public void testToList() {
    Assert.assertEquals(Arrays.asList("hello"), singleton.list("hello"));
  }

  @Test public void testToArray() {
    final Object[] a = singleton.array("hello");
    Assert.assertEquals("hello", a[0]);
    azzert.assertEquals(1, a.length);
  }
}

**Comment by yoavz1997** [comment]
There are still more places where Assert is used
![image](https://user-images.githubusercontent.com/33288016/48870747-96a00600-edea-11e8-9cf0-dd789108639c.png)
Some are harder to find because import static was used so it is found only once although the usage of Assert's functions is high

**Comment by Matanpugach** [comment]
Reviewing now.

**Comment by Matanpugach** [comment]
Yup, only 2 accurances are in azzert.java:
![image](https://user-images.githubusercontent.com/27849930/48898264-04811780-ee55-11e8-9bd1-e56c8e2254be.png)
Well done, @ofiralexi #79 👍
